src/App.tsx

```diff
@@ -1,24 +1,37 @@
 /* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import React, { useContext } from 'react';
 import { UserWarning } from './UserWarning';
-
-const USER_ID = 0;
+import { TodoContextProps } from './types/TodoContextProps';
+import { TodoContext } from './components/TodosContext';
+import { USER_ID } from './utils/constants';
+import { Header } from './components/Header';
+import { TodoList } from './components/TodoList';
+import { Footer } from './components/Footer';
+import { ErrorComponent } from './components/ErrorComponent';
 
 export const App: React.FC = () => {
+  const {
+    todos,
+  }:TodoContextProps = useContext(TodoContext);
+
   if (!USER_ID) {
     return <UserWarning />;
   }
 
   return (
-    <section className="section container">
-      <p className="title is-4">
-        Copy all you need from the prev task:
-        <br />
-        <a href="https://github.com/mate-academy/react_todo-app-add-and-delete#react-todo-app-add-and-delete">React Todo App - Add and Delete</a>
-      </p>
+    <div className="todoapp">
+      <h1 className="todoapp__title">todos</h1>
+
+      <div className="todoapp__content">
+        <Header />
+
+        {todos && <TodoList />}
```

Comment:
```suggestion
        {!!todos?.length && <TodoList />}
```
in rendering with &&  need to convert the value to boolean.